packages/web-components/@sfx/autocomplete/src/autocomplete.ts

```diff
@@ -156,14 +157,11 @@ export default class Autocomplete extends LitElement {
       .reduce((accItems, items) => [...accItems, ...items], []);
     const term = allItems[this.selectedIndex].label;
 
-    const sentEvent = new CustomEvent<AutocompleteActiveTermPayload>(AUTOCOMPLETE_ACTIVE_TERM, {
-      detail: {
-        query: term,
-        group: this.group,
-      },
-      bubbles: true,
-    });
-    this.dispatchEvent(sentEvent);
+    const payload: AutocompleteActiveTermPayload = {
+      query: term,
+      group: this.group,
+    };
+    super.dispatchSfxEvent(AUTOCOMPLETE_ACTIVE_TERM, payload);
```

Comment:
Given that `dispatchSelectedTerm()` has changed and now relies on `super.dispatchSfxEvent()`, its tests should change. Namely, take a look at any of the `expect()` assertions in the autocomplete tests inside of the `describe()` block for `dispatchSelectedTerm()`. They can be simpler and now just assert that the relevant `super` method is called with the right arguments.

Reply:
I somehow didn't notice this the first time through, but it should also be `this.dispatchSfxEvent`, not `super`. Because of inheritance, `dispatchSfxEvent` is available on `this`, and it's preferable to access it through the smallest scope possible.